server/config/store_config.go

```diff
@@ -194,17 +195,19 @@ func (c *StoreConfig) Clone() *StoreConfig {
 type StoreConfigManager struct {
 	config atomic.Value
```

Comment:
Is there possible that we move it to persist options?

Reply:
Good idea. Let me have a try.